crates/chain/Cargo.toml

```diff
@@ -13,18 +13,19 @@ readme = "../README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bitcoin = { version = "0.29" }
+bitcoin = { version = "0.29", default-features = false }
 serde_crate = { package = "serde", version = "1", optional = true, features = ["derive"] }
 
 # Use hashbrown as a feature flag to have HashSet and HashMap from it.
 # note version 0.13 breaks outs MSRV.
-hashbrown = { version = "0.12", optional = true,  features = ["serde"] }
-miniscript = { version = "9.0.0", optional = true  }
+hashbrown = { version = "0.12", optional = true, features = ["serde"] }
+miniscript = { version = "9.0.0", optional = true, default-features = false }
 
 [dev-dependencies]
 rand = "0.8"
 
 [features]
 default = ["std", "miniscript"]
-std = []
+std = ["bitcoin/std", "miniscript/std"]
```

Comment:
:x: I don't think we need `miniscript/std` for anything. This will unnecessarily add miniscript as a dependency if someone enables the `std` feature.